core/packages/ads/shared/components/Ad/AdSense/index.js

```diff
@@ -1,7 +1,12 @@
+/* eslint-disable react/no-danger */
 import React, { PureComponent, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { Helmet } from 'react-helmet';
+import Script from '../../Script';
+
+// eslint-disable-next-line
+import call from 'raw-loader!babel-loader?forceEnv=devClient!./functions/call';
```

Comment:
This loader is obsolete. Once https://github.com/frontity/frontity/pull/124 is merged, `dev` should be merged to this branch to fix bugs.